### PR TITLE
feat(graphql): introduce libplanet-explorer as `chainQuery'

### PR DESCRIPTION
**Pull request by moreal**
This pull request introduces `chainQuery` which equals with [libplanet-explorer](https://github.com/planetarium/libplanet-explorer), as a step of #298. This query will provide queries like below screenshot as libplanet-explorer does. And it will replace the other queries (e.g. `getTx`, `nodeStatus.tip`, etc) in next pull requests step by step.

image

## What this pull request changes

- Introduce new query `chainQuery`, libplanet-explorer's root `Query` as a sub-query.
- Remove `GraphType`s copied from libplanet-explorer or duplicated, to replace them with the `GraphType`s in libplanet-explorer.
  - `AddressType`
  - `ByteStringType`
  - `ActionType`

### CODE REVIEW

**Comment by moreal** [comment]
It doesn't work well because `ByteStringType` of libplanet-explorer has bug. So I opened [a pull request](https://github.com/planetarium/libplanet-explorer/pull/161) to fix it.

**Comment by moreal** [comment]
> It doesn't work well because `ByteStringType` of libplanet-explorer has bug. So I opened [a pull request](https://github.com/planetarium/libplanet-explorer/pull/161) to fix it.

The pull request, https://github.com/planetarium/libplanet-explorer/pull/161, was merged and I bumped.

File: NineChronicles.Headless/BlockChainContext.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+using Libplanet.Action;
+using Libplanet.Blockchain;
+using Libplanet.Explorer.Interfaces;
+using Libplanet.Store;
+using Nekoyume.Action;
+using NCAction = Libplanet.Action.PolymorphicAction<Nekoyume.Action.ActionBase>;
+
+namespace NineChronicles.Headless
+{
+    public class BlockChainContext : IBlockChainContext<NCAction>
```
Comment (ipdae) [inline NineChronicles.Headless/BlockChainContext.cs:10]: What's the difference between `BlockChainContext` and `StandAloneContext`? There seems to be a lot of duplicate arguments.

Comment (moreal) [reply NineChronicles.Headless/BlockChainContext.cs:10]: As you said, they have common features and `BlockChainContext` just act like view of `StandaloneContext`. Why I made `BlockChainContext` was that `Libplanet.Explorer.Queries.Query` needs it.

https://github.com/planetarium/libplanet-explorer/blob/553cbf05d61eff4ab4d5c408768136225dc8f5cf/Libplanet.Explorer/Queries/Query.cs#L23-L35

**Review by ipdae — APPROVED** [review]

**Comment by moreal** [comment]
Could you review this pull request? @longfin @area363

**Review by longfin — APPROVED** [review]

**Review by area363 — APPROVED** [review]

**Pull request merged**